source/includes/usage-examples/read-code-examples.java

```diff
@@ -0,0 +1,59 @@
+// start-find-one
+FindPublisher<Document> findDocPublisher = collection
+        .find(eq("<field name>", "<value>")).first();
+
+Flux.from(findDocPublisher)
+        .doOnNext(System.out::println)
+        .blockLast();
+// end-find-one
+
+// start-find-multiple
+FindPublisher<Document> findDocPublisher = collection
+        .find(eq("<field name>", "<value>"));
+        
+Flux.from(findDocPublisher)
+        .doOnNext(System.out::println)
+        .blockLast();
+// end-find-multiple
+
+// start-count-collection
+Publisher<Long> countPublisher = collection.countDocuments();
+
+Flux.from(countPublisher)
+        .doOnNext(System.out::println)
+        .blockLast();
+// end-count-collection
+
+// start-count-documents
+Publisher<Long> countPublisher = collection.countDocuments(
+           eq("<field name>", "<value>"));
+
+Flux.from(countPublisher)
+        .doOnNext(System.out::println)
+        .blockLast();
+// end-count-documents
+
+// start-estimate-count
+Publisher<Long> countPublisher = collection.estimatedDocumentCount();
+
+Flux.from(countPublisher)
+        .doOnNext(System.out::println)
+        .blockLast();
+// end-estimate-count
+
+// start-retrieve-distinct
+DistinctPublisher<String> distinctPublisher = collection.distinct(
+           "<field name>", "<type.class>");
```

Comment:
Note to reviewer: I was unsure how to write type.class as a placeholder value. An example of type.class would be "String.class", but this would not be in quotation marks. However, I think doing it without quotation marks confuses the automatic coloring in VSCode. Would love to get your opinion on how I would write this as a placeholder value. Thanks!

Reply:
It is ok if the formatting isnt correct, it often misses things. 

I think this could be
```java
DistinctPublisher<String> distinctPublisher = collection.distinct(
           "<field name>", <type>.class);
```